README.md

```diff
@@ -127,8 +127,9 @@ Link to extension: https://github.com/civicrm/org.civicrm.casetokens
 * Case status now has a ‘Status class’ field to show cases which are incomplete, completed, or cancelled.
 
 # Technical Requirements
-* CiviCRM: version >= v5.8, preferably the latest version.
+* CiviCRM: version 5.35.
```

Comment:
@deb1990  will CiviCase extension work with version 5.35 community version?  or Does it require Compucorp fork version? I think it requires Compucorp's fork version, we should add the description and link to our tarball/release page too.